### PR TITLE
Kelsha Darby - User Story 29

### LINKED ISSUE #32: User Story 29, User views an Order Show Page
As a registered user
When I visit my Profile Orders page
And I click on a link for order's show page
My URL route is now something like "/profile/orders/15"
I see all information about the order, including the following information:
- the ID of the order
- the date the order was made
- the date the order was last updated
- the current status of the order
- each item I ordered, including name, description, thumbnail, quantity, price and subtotal
- the total quantity of items in the whole order
- the grand total of all items for that order

### PR DESCRIPTION
**Pull request by kelshadarby**
## A registered user can view an orders show page
Closes #32 

**_Test_**
- Created `spec/features/orders/show_spec`
    - This could also be moved to a nested orders folder under user

**_Routes_**
- Add a GET route to `users/orders/:order_id`

**_Controller_**
- Add a show method that finds an order using the params

**_View_**
- Create the order show page with the following information 
    - Order ID
    - Date Created
    - Date Updated
    - Order Status
    - Each Item Ordered With the Following Info
          -Item Name
          -Item Description
          -Item Image
          -Item Quantity
          -Item Price
          -Item Subtotal
    - Total Quantity of Items
    - Grand Total

### CODE REVIEW

File: app/views/user/orders/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+</section>
+<section id="status">
+  <p>Status: <%=@order.status.capitalize()%></p>
+</section>
+<section id="datecreated">
+  <p> <%= @order.created_at%></p>
+</section>
+<section id="dateupdated">
+  <p> <%= @order.updated_at%></p>
+</section>
```
Comment (brian-greeson) [inline app/views/user/orders/show.html.erb]: The id's here are a bit inconsistent. The first one is kabab case and all the rest are combined together.

**Review by brian-greeson — CHANGES_REQUESTED** [review]
Looks pretty good 2 small things
- The id's for the sections at the bottom of the view are inconsistent
- Why are you setting id's if you are not targeting them in the test?

File: app/views/user/orders/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <td><p><%= number_to_currency(item_order.price)%></p></td>
+        <td><p><%= item_order.item.image %></p></td>
+        <td><p><%= item_order.quantity%></p></td>
+        <td><p><%= number_to_currency(item_order.subtotal)%></p></td>
+      </section>
+    </tr>
+  <% end %>
+</table>
+
+<section id="total-quantity">
```
Comment (fredrondina) [inline app/views/user/orders/show.html.erb:27]: once the section ID's are normalized this will be good to go :)

**Review by fredrondina — APPROVED** [review]

File: spec/features/orders/show_spec.rb [hunk 3]
```diff
@@ -0,0 +1,85 @@
+require "rails_helper"
```
Comment (brian-greeson) [inline spec/features/orders/show_spec.rb]: I think this is in the wrong test file? should be in /users/orders/show_spec.rb

**Review by brian-greeson — APPROVED** [review]
Looks good.

**Pull request merged**